virttest/utils_test/libvirt.py

```diff
@@ -1592,6 +1592,22 @@ def alter_boot_order(vm_name, pci_id, boot_order=0):
     vmxml.sync()
 
 
+def wait_for_file_over(patt, xml_file, timeout=10):
+    """
+    Wait for file write over with pattern
+
+    :param patt: pattern for grep in file to wait, eg '</disk>'
+    :param xml_file: file for writing xml
+    :param timeout: wait_for timeout
+    """
+
+    def file_write_over():
+        if not process.run("grep %s %s" % (patt, xml_file),
+                           ignore_status=True).exit_status:
+            return True
+    utils_misc.wait_for(file_write_over, timeout)
```

Comment:
Are we considering returning what `utils_misc.wait()` returns?

Reply:
I think as it just wait for the file write finish, there should be no need to do judgement when calling it.

Reply:
OK, I understand. Thanks for explaining.